packages/tools/kadena-cli/src/account/utils/accountHelpers.ts

```diff
@@ -145,3 +174,48 @@ export const fundAmountValidation = z
   })
   .min(1)
   .max(100);
+
+export const getChainIdRangeSeparator = (
+  input: string,
+): ',' | '-' | undefined => {
+  const hasComma = input.includes(',');
+  const hasHyphen = input.includes('-');
+  return hasComma ? ',' : hasHyphen ? '-' : undefined;
+};
+
+const getChainIds = (
+  input: string,
+  separator?: ',' | '-',
+): number[] | undefined => {
+  if (isEmpty(separator)) {
+    return [parseInt(input.trim(), 10)];
+  } else if (separator === ',') {
+    const splitValue = new Set(input.split(','));
+    return [...splitValue]
+      .map((id) => parseInt(id.trim(), 10))
+      .filter(notEmpty);
```

Comment:
parseInt failing will result in `NaN`, I don't know if `notEmpty` filters that correctly

Reply:
It doesn't filter out the `NaN` it just returns the values and i'm validating against ZodSchema here https://github.com/kadena-community/kadena.js/blob/feat/support-multi-chain-ids/packages/tools/kadena-cli/src/prompts/account.ts#L275-L281

Reply:
<img width="1264" alt="image" src="https://github.com/kadena-community/kadena.js/assets/7163943/e4b76051-c431-4715-90b6-4121bbb83364">